### PR TITLE
[#243] Add search and pagination (connects #243)

### PR DESCRIPTION
**Pull request by Pr0chin**
Pagination section is left. so this issue is still in progress.

### CODE REVIEW

File: src/app/resources/resources.component.html [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+</div>
+
+<div class="mat-paginator-container" [ngStyle]="{'justify-content': 'flex-start'}">
+ <mat-paginator [pageSize]="pageSize" [pageSizeOptions]="pageSizeOptions" (page)="pageEvent = $event"></mat-paginator>
+</div>
+
 <ng-template ngFor let-item [ngForOf]="resources">
   <!-- Used ng-template insted of spans or divs because I did not want any to add any element -->
-  <div class="list-item">
+  <div class="list-item" *ngIf="item.doc.title.toLowerCase().includes(searchResource.toLowerCase())" >
```
Comment (lmmrssa) [inline src/app/resources/resources.component.html]: material has its own filter for filtering data from record array so use that

Comment (Pr0chin) [reply src/app/resources/resources.component.html]: well I m not able to use material features as filter is use ur mat-table..as it has not been use in resource.

File: src/app/resources/resources.component.html [hunk 2]
```diff
@@ -18,10 +30,10 @@ <h3>{{ item.doc.title }}</h3>
         </ng-template>
       </div>
       <div class="list-item-action">
-        <a mat-raised-button i18n [routerLink]="['/resources/view', item.doc._id]">View Details</a>
-        <a mat-raised-button i18n [routerLink]="['/resources/wip']">Add To Library</a>
-        <a mat-raised-button i18n [routerLink]="['/resources/wip']">Feedback</a>
-        <a mat-raised-button i18n [routerLink]="['/resources/wip']">Delete</a>
+        <a mat-raised-button color = "primary" i18n [routerLink]="['/resources/view', item.doc._id]">View Details</a>
```
Comment (nirojdyola) [inline src/app/resources/resources.component.html]: It should be color="primary" , no space . Could you please check it once ?

**Comment by Pr0chin** [comment]
Had made reference branch #250 .. so Guys you can go through it :+1:

**Pull request closed without merging**